integration/monai/examples/spleen_ct_segmentation_real-world/job_he/app/config/config_fed_server.json

```diff
@@ -0,0 +1,72 @@
+{
+  "format_version": 2,
+
+  "min_clients": 2,
+  "num_rounds": 100,
+
+  "task_data_filters": [],
+  "task_result_filters": [],
+  "components": [
+    {
+      "id": "persistor",
+      "path": "monai_nvflare.monai_bundle_persistor.MonaiBundlePersistor",
+      "args": {
+        "bundle_root": "config/spleen_ct_segmentation"
```

Comment:
Does this need to change to "spleen_ct_segmentation_real-world"?

Reply:
No, bundle_root is relative to the app_root.